spec/controllers/admin_outgoing_message_controller_spec.rb

```diff
@@ -55,6 +55,36 @@
 
     end
 
+    context 'unsuccessfully destroying the message' do
```

Comment:
"unsuccessfully destroying" seems odd is all; maybe "unsuccessful attempt at destroying" (or "to destroy")